### PR TITLE
PRODSEC-1192 add jira flow

### PR DESCRIPTION
**Pull request by sfc-gh-jfan**

### CODE REVIEW

**Comment by sfc-gh-jfan** [comment]
we are now using a new jira token for all public repos to create security vulnerability tickets reported by whitesource and we have informed repo owner to change the configuration on forking pull request from third party.

File: .github/workflows/whitesource.yml [hunk 1]
```diff
@@ -24,6 +24,6 @@ jobs:
     - name: PR
       uses: ./whitesource-actions/whitesource-pr
       with:
-        jira_token: ${{ secrets.JIRA_TOKEN }}
+        jira_token: ${{ secrets.JIRA_TOKEN_PUBLIC_REPO }}
```
Comment (sfc-gh-japatel) [inline .github/workflows/whitesource.yml:27]: I am assuming this is already added somewhere?

Comment (sfc-gh-japatel) [reply .github/workflows/whitesource.yml:27]: never mind i see those in secrets of this repo inherited from parent (snowflakedb)

**Review by sfc-gh-japatel — APPROVED** [review]
lgtm

**Pull request merged**